npm error

`npm install hooverd`

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/hooverd - Not found
npm ERR! 404 
npm ERR! 404  'hooverd@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/steve/.npm/_logs/2019-11-07T15_27_37_285Z-debug.log
```

--- a/hooverd
+++ b/hooverd
@@ -1,0 +1,3 @@
+#!/usr/bin/env node
+const startServer = require('./hooverd.js');
+startServer();

--- a/hooverd.js
+++ b/hooverd.js
@@ -127,7 +127,9 @@ async function dispatchTX(tx, response) {
     response.end(output + "\n")
 }
 
-async function startServer() {
+module.exports = async function startServer() {
+    console.log("Welcome to hooverd! 👋\n\nWe are...")
+    
     // Print introductory information to the console.
     console.log(`...starting a server at http://localhost:${port}.`)
 
@@ -152,5 +154,3 @@ async function startServer() {
 
 }
 
-console.log("Welcome to hooverd! 👋\n\nWe are...")
-startServer()

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "hooverd",
-  "version": "1.0.1",
+  "version": "1.0.2",
   "description": "A simple server for Arweave transaction signing and dispatch.",
   "main": "hooverd.js",
   "dependencies": {

--- a/package.json
+++ b/package.json
@@ -17,6 +17,9 @@
     "dweb",
     "signing"
   ],
+  "bin": {
+    "hooverd": "hooverd"
+  },
   "author": "samcamwilliams",
   "license": "GPL-2.0-only"
 }

